touch_element/test_apps/main/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+idf_component_register(SRCS "test_app_main.c"
+                            "test_touch_element.c"
+                            "test_touch_button.c"
+                            "test_touch_slider.c"
+                            "test_touch_matrix.c"
+                       INCLUDE_DIRS "../../include_priv"
+                       REQUIRES "touch_element" "unity"
```

Comment:
Are you sure that this actually compile and links the `touch_element` from this repository?
It might link to `touch_element` from esp-idf. (But I need to double-check)

Reply:
Nice catch! I missed this point, I'll test it. BTW, do we have any way to run the CI locally? So that I can reduce the retry times on CI.

Reply:
Since you need only a generic runner, you have to do only the following:
1. cd to your touch_element test_app
2. Build for whichever target you want (eg. esp32s2)
3. Run `pytest --target=esp32s2`

(You need to have pytest installed on your local machine, 
```
pip install pytest-embedded pytest-embedded-serial-esp pytest-embedded-idf
```
)